packages/jest-cli/src/TestPathPatternPrompt.js

```diff
@@ -107,29 +96,32 @@ module.exports = (
             const filePath = trimAndFormatPath(padding, config, rawPath, width);
             return highlight(rawPath, filePath, pattern, config.rootDir);
           })
-          .forEach(
-            filePath => pipe.write(`\n  ${chalk.dim('\u203A')} ${filePath}`)
-          );
+          .forEach(filePath =>
+            pipe.write(`\n  ${chalk.dim('\u203A')} ${filePath}`));
```

Comment:
This is the first time I see people write code like this. The rule we encoded inside of prettier is that if the last argument is a function, then we are going to put the arguments in the first line and then the body indented in the next line.

Can you explain when you don't want the behavior I just explained above in this case? Is there a way we can generalize it?